SentryTestUtils/ClearTestState.swift

```diff
@@ -26,9 +26,6 @@ class TestCleanup: NSObject {
         setTestDefaultLogLevel()
 
         #if os(iOS) || os(tvOS) || targetEnvironment(macCatalyst)
-        let framesTracker = SentryDependencyContainer.sharedInstance().framesTracker
```

Comment:
`SentryDependencyContainer.reset` now stops the frames tracker, which this method also calls.